neqo-transport/src/connection/mod.rs

```diff
@@ -225,6 +225,69 @@ impl AddressValidationInfo {
     }
 }
 
+#[derive(Clone)]
```

Comment:
This is small now, but it is going to grow.  Can you put this in a separate file?  You can `pub use` it so that it is available without the extra long name.